docs/docs/upgrading.md

```diff
@@ -19,6 +19,11 @@ The `signing_key_algorithm` option has been removed and will now be inferred fro
 ### Changed GitHub Team IDs
 To improve performance, IdP directory synchronization for GitHub now uses the GraphQL API. This API returns the same information as the REST API, except that the GraphQL node IDs are different. Where we previously used the team integer ID from the REST API, we now use the team slug instead. Most policies should already use the team slug for group based rules, which should continue to work. However, if the integer ID is used it will no longer work. Update those policies to use the team slug instead.
 
+### CLI Source and Packaging Update
+`pomerium-cli` has been factored out of the core repository and now resides at <https://github.com/pomerium/cli>. If you currently install the CLI tool from [Packages](docs/releases.md#packages) or [Homebrew](/docs/releases.md#homebrew), no changes should be required to your process. However, users of docker images or direct github release downloads will need to update their references.
```

Comment:
I can't comment directly on [unedited lines](https://github.com/pomerium/pomerium/pull/2790/files#diff-52bfd0219348458c0d8cbcadfe4f8f9d0f1854bc1dd7c3a9b4e558f65e77e872R146), but should the package instructions be updated to point to `pomerium/cli` instead of `pomerium/pomerium`?

Reply:
That's still accurate.  We're sharing the package repository - they were always two separate packages.